src/bindings/haskell/libelektra-haskell.cabal.in

```diff
@@ -0,0 +1,40 @@
+name:                libelektra-haskell
```

Comment:
Is this file a duplicate of  src/bindings/haskell/libelektra-haskell.cabal?
Is  src/bindings/haskell/libelektra-haskell.cabal really needed?

Reply:
no it is not needed, it's just the cabal file used for my local development purposes, excluded it.